package.json

```diff
@@ -12,7 +12,6 @@
     "svg": "cd src/svgs/ && node generate-svg-sprite.js && cd ../../ && node copy-svgs.js",
     "prod": "node clean-prod.js && ng build --aot --prod --build-optimizer --output-path=prod/ --delete-output-path=false && node copy-index.js",
     "push-prod": "cd prod && git add . && git commit -m \"build(slpdev): release\" && git push",
-    "release": "npm run prod && npm run push-prod",
```

Comment:
@EyeOfPython Please keep this one :)

Reply:
Oops! Was totally intentional to check if you're actually reviewing ;)

Reply:
No worries :)